server/libs/userlibs.js

```diff
@@ -0,0 +1,52 @@
+const {User} = require('../DataBase/models/user');
+const _ = require('lodash');
+const fs = require('fs');
+const bcrypt = require('bcryptjs');
+const validator = require('validator');
+const formidable = require('formidable');
+
+//register a new user to the app
+exports.register = function(req,res) {
+    let body = _.pick(req.body, ['name', 'email','password']);
+    let user = new User(body);
```

Comment:
this is not a good way import model and then use that model

Reply:
Node.js doesn't support ES6's import yet. So that's why used require